README.developer.md

```diff
@@ -387,3 +388,7 @@ subdirectory:
 ```bash
 SKIP_BUILDDEP=yes ARTIFACTS_DIR=${PWD}/output ./build-scripts/build-rpm.sh
 ```
+
+### Debian
```

Comment:
Wouldn't it be better to use `dpkg` instead of `Debian`? Using `dpkg` aligns with existing usage of `RPM` term ...

Reply:
Makes sense. Changed it :heavy_check_mark: